### ISSUE TITLE
Deprecate Keep original screenshot format option

### ISSUE THREAD

**Issue by shadowpanther** [Yellow-Dog-Man/Resonite-Issues#1878]
### Is your feature request related to a problem? Please describe.

After #1877 is implemented, and considering that we already have #870, the Keep original screenshot format option is not needed anymore, as we would have options both for in-game saving format and hard drive saving format.

Additionally, the current wording for this option's description is confusing.

### Describe the solution you'd like

Deprecate and remove the Keep original screenshot format option in Misc.

### Describe alternatives you've considered

Keep the current mess of two options in different categories that do weird things in combination? I dunno.

**Comment by shiftyscales** [comment]
Per our [reporting requirements](https://github.com/Yellow-Dog-Man/Resonite-Issues#reporting-requirements):
> - **Use objective, and respectful language** for any posts/comments submitted (negativity, hateful/angry issues/comments are liable to be edited or removed)
> - **Focus on the high level issue first, not specific solution** as this lets us approach the problem in most efficient way. E.g. it's better to create issue saying "I can't do X", rather than "Add component that when this and this it does this". Often there are many different solutions to a problem and if you ask us to implement a very specific solution, instead about asking the problem you're trying to solve, you're more likely to get a lot of questions and possibly no. To learn more, check this page about the XY Problem: https://xyproblem.info/

This point would be better raised in your other issue, #1877. There is use for this setting as-is, and I don't think it should be removed.

> the Keep original screenshot format option is not needed anymore

While it has some overlap with being able to manually export images to disk in a chosen format, it doesn't supersede/replace the function and purpose of keeping the photo in its original encoding format. If this setting were removed, the only way you could keep the original format would be to know what format it was encoded in, and change your export encoding setting to match.

I think a better solution would be to combine the two. Adding 'keep original format' as an option for the export format.

**Comment by shadowpanther** [comment]
> This point would be better raised in your other issue, #1877. There is use for this setting as-is, and I don't think it should be removed.

What happened to "make a new issue for each separate problem"?

> While it has some overlap with being able to manually export images to disk in a chosen format, it doesn't supersede/replace the function and purpose of keeping the photo in its original encoding format. If this setting were removed, the only way you could keep the original format would be to know what format it was encoded in, and change your export encoding setting to match.

Adding an option to set the export format as WEBP supersedes the "keep original" as the original format _is_ WEBP. See Froox's comment here: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/870#issuecomment-1849130744

**Comment by Banane9** [comment]
> Adding an option to set the export format as WEBP supersedes the "keep original" as the original format _is_ WEBP. See Froox's comment here: [#870 (comment)](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/870#issuecomment-1849130744)

In which he also explains, that the format setting should set how a screenshot is _exported_, rather than how it is saved in-game.

**Comment by shiftyscales** [comment]
> What happened to "make a new issue for each separate problem"?

This is a part of the same problem, hence why I labelled it as a duplicate- and your option to remove this setting outright is not a solution to the issue you presented, because this setting solves a different problem/use-case.

> Adding an option to set the export format as WEBP supersedes the "keep original" as the original format is WEBP. See Froox's comment here

This is no longer the case- now it is whatever format you set it to in the encode settings (for the finger photo), or whatever the `InteractiveCamera` component specifies as I mentioned in #1879, depending on the camera, @shadowpanther.

@Banane9 - he was explaining that "Keep original screenshot format" is controlling the export encoding format. He was not talking about the quick photo capture encoding format setting. In other words- he was talking about what #1877 is requesting.

**Comment by Banane9** [comment]
> he was explaining that "Keep original screenshot format" is controlling the export encoding format. He was not talking about the quick photo capture encoding format setting. In other words- he was talking about what #1877 is requesting.

He was talking about how to implement #870 - namely by adding an option to determine the format in which pictures are exported.
